tests/test_operations.rs

```diff
@@ -35,6 +35,20 @@ fn test_op_log() {
     o 56b94dfc38e7 test-username@host.example.com 2001-02-03 04:05:07.000 +07:00 - 2001-02-03 04:05:07.000 +07:00
       initialize repo
     "###);
+    // Test op log with relative dates
+    let stdout = test_env.jj_cmd_success(
+        &repo_path,
+        &["op", "log", "--config-toml", "ui.relative-timestamps=true"],
+    );
+    insta::assert_snapshot!(&stdout, @r###"
+    @ 45108169c0f8 test-username@host.example.com 21 years ago - 21 years ago
+    | describe commit 230dd059e1b059aefc0da06a2e5a7dbf22362f22
+    | args: jj describe -m 'description 0'
+    o a99a3fd5c51e test-username@host.example.com 21 years ago - 21 years ago
+    | add workspace 'default'
+    o 56b94dfc38e7 test-username@host.example.com 21 years ago - 21 years ago
+      initialize repo
+    "###);
```

Comment:
This would need to be updated once a year. That's not terrible, but it's a bit annoying, so let's replace "\d+ years ago" by "N years ago" or something like that.